en/micro-integrator/docs/references/connectors/fhir-connector/fhir-connector-example.md

```diff
@@ -0,0 +1,318 @@
+                   <format>{$ctx:format}</format>
+                   <summary>{$ctx:summary}</summary>
+               </fhir.readSpecificResourceById>
+               <log level="full" separator=","/>
+               <send/>
+           </inSequence>
+           <outSequence/>
+           <faultSequence/>
+       </resource>
+       <resource methods="POST" url-mapping="/delete">
+           <inSequence>
+               <property expression="json-eval($.base)" name="base" scope="default" type="STRING"/>
+               <property expression="json-eval($.resourceType)" name="type" scope="default" type="STRING"/>
+               <property expression="json-eval($.format)" name="format" scope="default" type="STRING"/>
+               <property expression="json-eval($.idToDelete)" name="idToDelete" scope="default" type="STRING"/>
+               <fhir.init>
+                   <base>{$ctx:base}</base>
+               </fhir.init>
+               <fhir.delete>
+                   <type>{$ctx:type}</type>
+                   <idToDelete>{$ctx:idToDelete}</idToDelete>
+               </fhir.delete>
+               <log level="full" separator=","/>
+               <send/>
+           </inSequence>
+           <outSequence/>
+           <faultSequence/>
+       </resource>
+       <resource methods="POST" url-mapping="/update">
+           <inSequence>
+               <property expression="json-eval($.base)" name="base" scope="default" type="STRING"/>
+               <property expression="json-eval($.resourceType)" name="type" scope="default" type="STRING"/>
+               <property expression="json-eval($.format)" name="format" scope="default" type="STRING"/>
+               <property expression="json-eval($.idToUpdate)" name="idToDelete" scope="default" type="STRING"/>
+               <fhir.init>
+                   <base>{$ctx:base}</base>
+               </fhir.init>
+               <fhir.update>
+                   <type>{$ctx:type}</type>
+                   <idToUpdate>{$ctx:idToUpdate}</idToUpdate>
+                   <format>{$ctx:format}</format>
+               </fhir.update>
+               <log level="full" separator=","/>
+               <send/>
+           </inSequence>
+           <outSequence/>
+           <faultSequence/>
+       </resource>
+   </api>
+
+    ```
+3. Now we can export the imported connector and the API into a single CAR application. CAR application is the one we are going to deploy to server runtime.
+
+{! /references/connectors/exporting-artifacts.md !}
+
+## Deployment
+
+Follow these steps to deploy the exported CApp in the Enterprise Integrator Runtime. 
+
+{!references/connectors/deploy-capp.md!}
+    
+## Testing
+
+Invoke the API as shown below using the curl command. Curl Application can be downloaded from [here](https://curl.haxx.se/download.html).
+
+1. Create resource operation.
+   
+   **Sample Request** 
+      
+   ```
+   curl -v POST -d 
+   '{
+      "base": "http://hapi.fhir.org/baseR4",
```

Comment:
Should user provide "base"? We can hardcode it. 
What are the allowed formats? Can we inject it using `Content-Type` (using switch mediator we can set the property).

Reply:
If user need to use another FHIR test server or API for their scenarios they should provide base URL as this parameter.

Allowed formats of the request should be the Mime Type.

Reply:
We should add a note on this.

Reply:
- Added switch mediator and update the sample API source. 
- While hardcoding the resourceType parameter and invoking the API requests, following issue occurred.
https://github.com/wso2/micro-integrator/issues/1350